docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,20 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5] | [1.6] | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|-------|-------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0   | 1.0   | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1 | 4.6.1 | 4.6.1 |
```

Comment:
Did you see how I marked out the versions in the .NET Framework row in my PR? that is because the tools are changing and what you have here represents tools that have not released yet. I would suggest we mark out the versions here and add a note about the tools change so folks at least have the information.